packages/common/src/schemas/integrations/zendesk/zendesk.schema.ts

```diff
@@ -0,0 +1,48 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import { Static, Type, querySyntax } from '@feathersjs/typebox'
+
+export const zendeskAuthenticationPath = 'zendesk-authentication'
+
+export const zendeskAuthenticationMethods = ['create'] as const
+
+export const zendeskAuthenticationSchema = Type.Object({
+  scope: Type.String(),
+  external_id: Type.Optional(Type.String()),
```

Comment:
We use camelCase Notation. Update to `externalId`

Reply:
No, zendesk uses snake case, if I use camelCase they won't find the user id in the token.
![image](https://github.com/EtherealEngine/etherealengine/assets/33644205/591c3516-0da2-49e9-9544-f17075e808be)